docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,555 @@
+---
+title: "Build a bedtime songs bot with a custom ML model"
+linkTitle: "Bedtime Songs Bot"
+weight: 60
+type: "docs"
+tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
+description: "Create a robot babysitter with a webcam and machine learning."
+webmSrc: "/tutorials/img/bedtime-songs-bot/robot_babysitter.webm"
+mp4Src: "/tutorials/img/bedtime-songs-bot/robot_babysitter.mp4"
+videoAlt: "A demonstration of the bedtime songs bot is taking place in an office. Tess holds up brightly colored puzzle pieces in front of the camera of a Macbook laptop. As the webcam on the laptop recognizes the puzzle pieces, different songs start to play on the speakers of the computer."
+authors: [ "Tess Avitabile", "Sierra Guequierre" ]
+languages: [ "python" ]
+viamresources: [ "camera", "sensor", "mlmodel", "vision" ]
+level: "Beginner"
+date: "21 April 2023"
+cost: "0"
+---
+
+## Author's Note
+
+**"When I started at Viam, Eliot Horowitz told me the best way to test the product is to try to automate something I do in my life with a robot.
+As a parent of a 3-year-old and a 1-year-old, I am often presented with a toy and asked to sing a song about it.
+When I was testing out Viam's ML Model service, I came up with the idea of using machine learning to make my computer do this simple task for my kids when I'm not around."**
+
+- Tess, Engineering Director at Viam
```

Comment:
This looks a bit strange. How about:
```suggestion
{{< alert title="Author's note" color="note" >}}

"When I started at Viam, Eliot Horowitz told me the best way to test the product is to try to automate something I do in my life with a robot.
As a parent of a 3-year-old and a 1-year-old, I am often presented with a toy and asked to sing a song about it.
When I was testing out Viam's ML Model service, I came up with the idea of using machine learning to make my computer do this simple task for my kids when I'm not around."

<p style="text-align: right;">Tess, Engineering Director</p>
{{< /alert >}}
```

Reply:
Done, thank you greatly for taking the time to put together a better format-- I couldn't figure it out